pkg/apis/kops/validation/validation.go

```diff
@@ -1248,8 +1248,8 @@ func validateNetworkingCilium(cluster *kops.Cluster, v *kops.CiliumNetworkingSpe
 			allErrs = append(allErrs, field.Invalid(versionFld, v.Version, "Could not parse as semantic version"))
 		}
 
-		if version.Minor != 16 {
-			allErrs = append(allErrs, field.Invalid(versionFld, v.Version, "Only version 1.16 is supported"))
+		if version.Minor != 17 {
+			allErrs = append(allErrs, field.Invalid(versionFld, v.Version, "Only version 1.17 is supported"))
```

Comment:
Do we need to hard-deprecate 1.16 for any reason?  I guess this is what we were doing before, but it seems quite aggressive...

Reply:
Cilium requires an updated template with every release, so we either have N large templates or we update for all releases.

Reply:
See the header of [k8s-1.16-v1.15.yaml.template]:(https://github.com/kubernetes/kops/blob/master/upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.16-v1.15.yaml.template)
```sh
helm template --release-name cilium cilium/cilium \
--version 1.16.2 \
--namespace kube-system \
--values helm-values.yaml
```

Reply:
@ameukam did you update the template ^^^?

Reply:
I did a partial update fix the CI but I'm not sure about a full update. the diff is pretty big.